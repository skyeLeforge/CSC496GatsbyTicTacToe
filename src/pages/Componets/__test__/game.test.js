import React from "react"
import { render } from "@testing-library/react"
import renderer from "react-test-renderer"

import Game from "../game.js"

//check to see if board renders correctly
describe("Game", () =>
  it("renders correctly", () => {
    const tree = renderer.create(<Game />).toJSON()
    expect(tree).toMatchSnapshot()
  })
)