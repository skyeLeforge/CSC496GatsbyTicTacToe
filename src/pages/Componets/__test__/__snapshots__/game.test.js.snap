// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game renders correctly 1`] = `
<div
  className="game"
>
  <div
    className="game-board"
  >
    <div>
      <div
        className="board-row"
      >
        <button
          className="square"
          data-testid="secret-test-id"
          onClick={[Function]}
        />
        <button
          className="square"
          data-testid="secret-test-id"
          onClick={[Function]}
        />
        <button
          className="square"
          data-testid="secret-test-id"
          onClick={[Function]}
        />
      </div>
      <div
        className="board-row"
      >
        <button
          className="square"
          data-testid="secret-test-id"
          onClick={[Function]}
        />
        <button
          className="square"
          data-testid="secret-test-id"
          onClick={[Function]}
        />
        <button
          className="square"
          data-testid="secret-test-id"
          onClick={[Function]}
        />
      </div>
      <div
        className="board-row"
      >
        <button
          className="square"
          data-testid="secret-test-id"
          onClick={[Function]}
        />
        <button
          className="square"
          data-testid="secret-test-id"
          onClick={[Function]}
        />
        <button
          className="square"
          data-testid="secret-test-id"
          onClick={[Function]}
        />
      </div>
    </div>
  </div>
  <div
    className="game-info"
  >
    <div>
      Next player: X
    </div>
    <ol>
      <li>
        <button
          onClick={[Function]}
        >
          Go to game start
        </button>
      </li>
    </ol>
  </div>
</div>
`;
